docs/events_handling.md

```diff
@@ -0,0 +1,102 @@
+
+# Events handling
+
+## Subscribed events
+
+Precaution needs to subscribe to a number of events
+to satisfy different use cases.
+
+The events are separated into two groups `pull requests
+events` and `check suites/runs events`.
+
+### Pull requests events
```

Comment:
I thought we'd agreed to use the event name from the GitHub docs, PullRequestEvent, as the section title?